src/views/ipv/page/confirm-your-details.njk

```diff
@@ -24,23 +24,46 @@
 
     <h1 class="govuk-heading-l" id="header" data-page="{{googleTagManagerPageId}}">{{ 'pages.confirmDetails.header' | translate }}</h1>
     <p class="govuk-body">{{ 'pages.confirmDetails.content.paragraph1' | translate }}</p>
-    <h3 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summaryHeading' | translate }}</h3>
+    <h2 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summaryHeading' | translate }}</h2>
 
-    <h3 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summary1' | translate }}</h3>
-    <p class="govuk-body">{{ userDetails.nameParts.givenName }}</p>
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-
-    <h3 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summary2' | translate }}</h3>
-    <p class="govuk-body">{{ userDetails.nameParts.familyName }}</p>
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-
-    <h3 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summary3' | translate }}</h3>
-    <p class="govuk-body">{{ userDetails.addresses[0].addressDetailHtml | safe }}</p>
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-
-    <h3 class="govuk-heading-s">{{ 'pages.confirmDetails.content.summary4' | translate }}</h3>
-    <p class="govuk-body">{{ userDetails.dateOfBirth | GDSDate }}</p>
-    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+    {% set rows = [
+      {
+        key: {
+          text: 'pages.confirmDetails.content.summary1' | translate
+        },
+        value: {
+          text: userDetails.nameParts.givenName
+        }
+      },
+      {
+        key: {
+          text: 'pages.confirmDetails.content.summary2' | translate
+        },
+        value: {
+          text: userDetails.nameParts.familyName
+        }
+      },
+      {
+        key: {
+          text: 'pages.confirmDetails.content.summary3' | translate
+        },
+        value: {
+          text: userDetails.addresses[0].addressDetailHtml | safe
+        }
+      },
+      {
+        key: {
+          text: 'pages.confirmDetails.content.summary4' | translate
+        },
+        value: {
+          text: userDetails.dateOfBirth | GDSDate
+        }
+      }
+    ] %}
+
+    {{ govukSummaryList({
+      rows: rows
+    }) }}
```

Comment:
(Slightly intrigued why we don't just inline the `rows` here, but it matches all our other usages, so better to be consistent!)